src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -43,6 +43,15 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
   const type = determineChatType(item);
   const isMe = checkIsMe(item, type);
 
+  const dbAnonUserInfo = item?.anon_user_info_color_code
+    ? {
+        anon_user_info_color_code: item?.anon_user_info_color_code,
+        anon_user_info_emoji_code: item?.anon_user_info_emoji_code,
+        anno_user_info_color_name: item?.anon_user_info_color_name,
+        anon_user_info_emoji_name: item?.anon_user_info_emoji_name
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the property name `anno_user_info_color_name`. It should be `anon_user_info_color_name` instead. This could lead to undefined behavior if not corrected.
```diff
-        anno_user_info_color_name: item?.anon_user_info_color_name,
+        anon_user_info_color_name: item?.anon_user_info_color_name,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->